### ISSUE TITLE
jquery/jquery.validate.min.js file isn't getting minified properly.

### ISSUE THREAD

**Issue by CGLOB** [tedious/JShrink#110]
[ pub/static/frontend/Magento/luma/en_US/jquery/jquery.validate.min.js ] is partially minified. It still contains parts without minifying.

Ex:
If you scroll to the bottom, you will see a non minified part.
[](url)
<img width="1503" alt="Screenshot 2023-01-04 at 2 21 03 pm" src="https://user-images.githubusercontent.com/87320997/222406135-3e321068-abb2-442a-9b2d-dea9504223ea.png">

**Comment by CGLOB** [comment]
I have solve under jetbrains://php-storm/navigate/reference?project=magento2ce&path=~/Projects/JShrink/src/JShrink/Minifier.php file. but not able to create PR due permission issue.

**Comment by tedivm** [comment]
What permission issue are you getting trying to make a PR? This repository is open for pull requests.

**Comment by tedivm** [comment]
Is this the same issue we discussed in the PRs? Is it okay to close it now?

**Comment by CGLOB** [comment]
> Is this the same issue we discussed in the PRs? Is it okay to close it now?

yes it is same issue. I am verifying with new version. If all good i will inform you the same.

**Comment by CGLOB** [comment]
Hi,
It is reproducible in latest version. Anyway i will create new PR with same solution along with test coverage as you suggested earlier. 
"To do that put a file named "regex_end_string.js" to-

./JShrink/tree/master/tests/Resources/jshrink/input/

And then add a second file with the expected output to-

./JShrink/tree/master/tests/Resources/jshrink/output/
"

**Comment by CGLOB** [comment]
Hi @tedivm any update about the PR?

**Comment by tedivm** [comment]
@CGLOB your PR has failing tests at the moment. If you can get those resolved I can give it a proper review.

**Comment by CGLOB** [comment]
> @CGLOB your PR has failing tests at the moment. If you can get those resolved I can give it a proper review.

I need your help. regex_close.js output is not correct. The output need to be modified. My solution is related regex_close.js code which suppose to minify fully. It should be "function test(string){return(string||'').replace(/([\\!"#$%&'()*+,./:;<=>?@\[\]^`{|}~])/g,'\\$1')}" 
and about Libraries:jquery_ui.js i am checking.

**Comment by CGLOB** [comment]
@tedivm any update about PR review?

**Comment by tedivm** [comment]
I'm brainstorming some adversarial strings- basically I'm just trying to beat this up some more to make sure there are no regressions. Regex is one of the most annoying parts of processing, and I've made changes in the past that I thought were great that added in regressions. As a result I'm super paranoid about changes to that part of the code base.

**Comment by fabiorachellojak** [comment]
Hello @tedivm we actually have the same problem, is there a way to fix that?
Thank you!
